traitsui/testing/tester/ui_wrapper.py

```diff
@@ -66,7 +66,7 @@ class UIWrapper:
         visual confirmation test code is working as desired.
     """
 
-    def __init__(self, target, registries, delay=0):
+    def __init__(self, target, registries, *, delay=0):
```

Comment:
I think there'd be benefit to making `registries` keyword-only, too, both for consistency with `UITester` and because it'll make the call sites more self-explanatory.

Reply:
SGTM!